pyiron_base/job/worker.py

```diff
@@ -188,21 +188,13 @@ def run_static_with_database(self):
                     & (~df["id"].isin(active_job_ids))
                 ]
                 if len(df_sub) > 0:  # Check if there are jobs to execute
-                    path_lst = [
-                        [pp, p, job_id]
-                        for pp, p, job_id in zip(
-                            df_sub["projectpath"].values,
-                            df_sub["project"].values,
-                            df_sub["id"].values,
-                        )
-                        if job_id not in active_job_ids
-                    ]
-                    job_lst = [
-                        [p, job_id] if pp is None else [os.path.join(pp, p), job_id]
-                        for pp, p, job_id in path_lst
-                    ]
-                    active_job_ids += [j[1] for j in job_lst]
```

Comment:
Are you missing the `active_job_ids` list? basically the jobs are not added to it, right?